README.md

```diff
@@ -19,6 +19,17 @@ This is provided via the `rust-systemd` crate, a Rust interface to `libsystemd/l
 To build the `runtime` make sure you have `libsystemd-dev` installed on your system
 and the systemd feature enabled.
 
+To install `libudev` on Debian-based Linux distributions, execute the following command:
```

Comment:
libudev development headers or something like that.

Reply:
Also I would expand this, with something like:
"libudev is required for gathering information about network interface, in order to install libudev development headers ..." or something like this.